### ISSUE TITLE
#50 broke our build

### ISSUE THREAD

**Issue by bootstraponline** [bitrise-io/steps-gradle-runner#51]
Is https://github.com/bitrise-io/steps-gradle-runner/pull/50 intended to be a breaking change?

**after**
```
    - gradle-runner:
        inputs:
        - gradle_task: -p "../teacher" :app:assembleQaDebugAndroidTest
        - gradlew_path: "./gradle/gradlew"
        - cache_level: all
        - gradle_options: "--stacktrace"
```

**before**

```
    - gradle-runner:
        inputs:
        - gradle_task: -p "teacher" :app:assembleQaDebugAndroidTest
        - gradlew_path: "./gradle/gradlew"
        - cache_level: all
        - gradle_options: "--stacktrace"
```

**error**

```
Running gradle task...
./gradlew "-p" "teacher" ":app:assembleQaDebugAndroidTest" "--stacktrace"
Unzipping /root/.gradle/wrapper/dists/gradle-4.1-all/bzyivzo6n839fup2jbap0tjew/gradle-4.1-all.zip to /root/.gradle/wrapper/dists/gradle-4.1-all/bzyivzo6n839fup2jbap0tjew
Set executable permissions for: /root/.gradle/wrapper/dists/gradle-4.1-all/bzyivzo6n839fup2jbap0tjew/gradle-4.1/bin/gradle
Starting a Gradle Daemon (subsequent builds will be faster)
FAILURE: Build failed with an exception.
* What went wrong:
The specified project directory '/bitrise/src/gradle/teacher' does not exist.
```

note that the path is `/bitrise/src/teacher` but the update changed it to `/bitrise/src/gradle/teacher`

- https://www.bitrise.io/build/9184d062bde98444

**Comment by viktorbenei** [comment]
Thanks for reporting @bootstraponline , we'll check this ASAP! :)

**Comment by kdobmayer** [comment]
Hi @bootstraponline 

We released a new version with the fix ([1.8.2](https://github.com/bitrise-io/steps-gradle-runner/releases/tag/1.8.2))

**Comment by bootstraponline** [comment]
Thanks!
